charts/div-ccd-definitions/values.preview.template.yaml

```diff
@@ -0,0 +1,65 @@
+tags:
+  ccd-idam-pr: true
+
+nodejs:
+  # Don't modify below here
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+
+ccd:
+  ingressHost: ${SERVICE_FQDN}
+  ingressIP: ${INGRESS_IP}
+  consulIP: ${CONSUL_LB_IP}
+
+  idam-pr:
+    releaseNameOverride: ${SERVICE_NAME}-ccd-idam-pr
+    redirect_uris:
+      CCD:
+        - https://case-management-web-${SERVICE_FQDN}/oauth2redirect
+      CCD Admin:
+        - https://admin-web-${SERVICE_FQDN}/oauth2redirect
+
+  apiGateway:
+    s2sKey: ${API_GATEWAY_S2S_KEY}
+    idamClientSecret: ${API_GATEWAY_IDAM_SECRET}
+
+  userProfileApi:
+    authorisedServices: ccd_admin,ccd_data,ccd_definition,cmc_claim_store
+
+  dataStoreApi:
+    s2sKey: ${DATA_STORE_S2S_KEY}
+    s2sAuthorisedServices: cmc_claim_store,ccd_gw
```

Comment:
do we need `divorce` s2s?

Reply:
I guess the value will just be `divorce`?